5.0/fr/0x17-V8-Authorization.md

```diff
@@ -0,0 +1,56 @@
+# V8 Autorisation
+
+## Objectif du contrôle
+
+L'autorisation garantit que l'accès est accordé uniquement aux utilisateurs autorisés (utilisateurs, serveurs et autres clients). Pour appliquer le principe du moindre privilège (POLP), les applications vérifiées doivent répondre aux exigences générales suivantes :
+
+* Règles d'autorisation des documents, y compris les facteurs décisionnels et les contextes environnementaux.
```

Comment:
" facteurs de décision" (or "facteurs de prise de décision") seems more lightweight to me.